src/frontend/pages/HistoryPage/HistoryPage.js

```diff
@@ -1,13 +1,50 @@
 import React from "react";
-import { useHistory } from "../../contexts";
+import { EmptyStatePage } from "../EmptyStatePage/EmptyStatePage";
+import { useAuth, useHistory } from "../../contexts";
 import "./HistoryPage.css";
+import { BigHorizontalVideoCard } from "../../components";
+import { removeAllHistoryService } from "../../services";
+import { SET_HISTORY } from "../../constants";
 
 export const HistoryPage = () => {
-  const { history } = useHistory();
+  const { history, dispatchHistory } = useHistory();
```

Comment:
Name something diferrent than `history`, react-router v5 have useHistory hook and Browser JS have history api which also named as `history` So, it would be recommanded ,  use othe rvariable name. (Just opinion)